lib/rust/ensogl/core/src/data/color/animation.rs

```diff
@@ -68,6 +68,8 @@ impl Animation {
                 |(color,alpha)| color.with_alpha(*alpha)
             );
         }
+        self.color_anim.precision.emit(0.01);
+        self.alpha_anim.precision.emit(0.01);
```

Comment:
Why this is needed? Make 0.01 a constant and add explanation why 0.01 and not another value (or default).